extract-metadata.py

```diff
@@ -0,0 +1,84 @@
+from dandi.dandiapi import DandiAPIClient
+from dandi.dandiapi import RemoteDandiset
+import pandas as pd
+from pathlib import Path
+
+modalities = {'oct': 'PS-OCT',
+              'df': 'Dark Field Microscopy',
+              'xpct': 'HiP-CT',
+              'dwi': 'DWI',
+              'fluo': 'LSM',
+              'photo': 'Blockface photo'}
+
+# Create dataframe of all assets across all datasets
+def extract_assets():
+    client = DandiAPIClient("https://api.lincbrain.org/api")
+    client.dandi_authenticate()
+
+    print(f"Processing {sum(1 for _ in client.get_dandisets())} Dandisets on lincbrain.org")
```

Comment:
`client.get_dandisets()` this is going to invoke this function that you later call just a few lines later -- you could simply call it once and store it as a variable such as `dandisets = client.get_dandisets()` and reference it later in your for-loop

Reply:
Thanks, updated.